src/containers/ReportArticle/state/helper.js

```diff
@@ -0,0 +1,55 @@
+export const reportPayload = {
```

Comment:
@SrMoffat you could name this file `mock.js` just to make it obvious that it is for mocking purposes.

Reply:
I have done so, the filename was refactored from `helpers` to `mock` as advised